css/style.scss

```diff
@@ -141,3 +141,7 @@ main p {
         transform: translateY(500px);
     }
 }
+
+.hidden {
+  display: none;
```

Comment:
instead of display: none I think a better trick might be:

```css
height: 0;
box-sizing: border-box;
overflow: hidden;
opacity: 0;
```

(sets height to 0 including padding, and opacity 0 just in case)

Because some bots might look for all visible form elements and fill those out (so display: none would exclude them).

But height 0 and opacity 0 results in a *technically* visible element to most implementations of checking visibility.